packages/flutter/test/painting/text_painter_rtl_test.dart

```diff
@@ -612,7 +612,7 @@ void main() {
           style: const TextStyle(fontSize: 10.0),
         ),
         const TextSpan(
-          text: ' ',
+          text: 'A',
```

Comment:
I'd rather we add a character, instead of replacing one of the spaces. For example as a third child with no `style`.
The test is checking the way spaces act.

Also we should add a comment that explains why the character is important.

Reply:
done